### PR TITLE
initialize transaction and minor changes.

### PR DESCRIPTION
**Pull request by lucas-zimerman**
* Ignore vscode settings.
* Add HttpMessageHandler to the server.
* Add initialization transaction.

### CODE REVIEW

File: game/Assets/Scripts/Game/StateMachine/GameStateMachine.cs [hunk 1]
```diff
@@ -11,16 +13,49 @@ public class GameStateMachine : StateMachine<GameStates>
...
         
-        _states.Add(GameStates.StartMenu, new GameStateStart(this));
-        _states.Add(GameStates.Upgrading, new GameStateUpgrading(this));
-        _states.Add(GameStates.SentryPlacing, new GameStatePlacing(this));
-        _states.Add(GameStates.Fighting, new GameStateFighting(this));
-        _states.Add(GameStates.GameOver, new GameStateGameOver(this));
+        using (new DisposableSpan(_startTransaction.StartChild("StartMenu", "state")))
+        {
+            _states.Add(GameStates.StartMenu, new GameStateStart(this));
+        }
```
Comment (bruno-garcia) [inline game/Assets/Scripts/Game/StateMachine/GameStateMachine.cs]: We're just measuring how long it takes to create an instance of `GameStateStart` and to add it to a list?

Comment (lucas-zimerman) [reply game/Assets/Scripts/Game/StateMachine/GameStateMachine.cs]: Yes, the time is quite low. Would you prefer to remove those spans and leave it only with the HTTP request span?

**Comment by lucas-zimerman** [comment]
https://sentry.io/organizations/sentry-sdks/discover/sentry-unity:261e48e2e5164c5da9ac763ec8e6ff65/?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&yAxis=count%28%29

The transaction.

EDIT:

new transactions

https://sentry.io/organizations/sentry-sdks/performance/summary/?project=6070139&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=1h&transaction=state.upgrading&unselectedSeries=p100%28%29

https://sentry.io/organizations/sentry-sdks/performance/summary/?project=6070139&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=1h&transaction=state.placing&unselectedSeries=p100%28%29

https://sentry.io/organizations/sentry-sdks/performance/summary/?project=6070139&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=1h&transaction=state.fighting&unselectedSeries=p100%28%29

https://sentry.io/organizations/sentry-sdks/performance/summary/?project=6070139&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=1h&transaction=state.gameover&unselectedSeries=p100%28%29

https://sentry.io/organizations/sentry-sdks/performance/summary/?project=6070139&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=1h&transaction=state.start&unselectedSeries=p100%28%29

round start with HTTP requests (whenever available and navigation breadcrumb)

https://sentry.io/organizations/sentry-sdks/performance/unity:b720e0a99ebf4d5ba02e15429adec285/?project=6070139&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=1h&transaction=round.start&unselectedSeries=p100%28%29

File: game/Assets/Scripts/Game/GameStateFighting.cs [hunk 2]
```diff
@@ -25,18 +25,25 @@ public GameStateFighting(GameStateMachine stateMachine) : base(stateMachine)
...
-        _frozenFrames = 0;
-        _totalFrames = 0;
+            _slowFrames = 0;
+            _frozenFrames = 0;
+            _totalFrames = 0;
 
-        base.OnEnter();
-        
-        bugsToSpawn = 5 + _data.Level * 2;
+            base.OnEnter();
```
Comment (bruno-garcia) [inline game/Assets/Scripts/Game/GameStateFighting.cs]: Does this belong here?

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**